src/components/Match/MatchLog.jsx

```diff
@@ -118,6 +119,69 @@ const logColumns = [heroTdColumn, {
           />
         );
       }
+      case 'objectives': {
+        if (row.detail === ' First Blood') {
```

Comment:
I suggest matching on a key value as display strings may change

Reply:
Unfortunately I can't find any key value I could use.

Reply:
If you look at the raw API data there is a type field:
https://api.opendota.com/api/matches/3584342151

Maybe it's not being passed down to this component it should be possible to do so